de.metas.swat/de.metas.swat.base/src/main/sql/postgresql/system/40-sw01_swat/5452310_mig_FRESH-783_Migrate_Old_Inouts_async.sql

```diff
@@ -0,0 +1,167 @@
+
+
+
+CREATE OR REPLACE FUNCTION create_request_document_number ()
```

Comment:
I suggest that you also prepend a `temp_` in front of this function, and add a [comment](https://www.postgresql.org/docs/9.1/static/sql-comment.html) to both the function or the table to indicate the github issue-URL and and that they are not required anymore after the migration finished.